vendor.conf

```diff
@@ -26,7 +26,7 @@ github.com/imdario/mergo                            7c29201646fa3de8506f70121347
 golang.org/x/sync                                   e225da77a7e68af35c70ccbf71af2b83e6acac3c
 
 # buildkit
-github.com/moby/buildkit                            b26cff2413cc6a466f8739262efa13bd126f8fc7 # v0.6.4 + aa7df97d7136e732561b59f87a38ad52d46d3b19
+github.com/moby/buildkit                            59e305aa33fd96e51d5c458f55104250b3e39f56 # v0.6.4-5-g59e305aa
```

Comment:
where's the `-5-` coming from?

Reply:
5 commits since the `v0.6.4` tag, and `59e305aa` is the commit:

```bash
git describe --tags --match  "v[0-9]*"

v0.6.4-5-g59e305aa
```